contrib/dind/test.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
I don't believe this worked with `/bin/sh` which is probably why it has bin/bash